Source/ProceduralPart.cs

```diff
@@ -1479,7 +1490,13 @@ public void PartParentChanged(Part newParent)
             // ReSharper disable once InconsistentNaming
             Func<Vector3> Offset;
             if (nodeOffsets.TryGetValue(childToParent.id, out Offset))
+            {
                 position -= Offset();
+            }
+            else if(srfAttached)
+            {
+                position -= (childToParent.attachedPart.transform.position - childToParent.owner.transform.position);
```

Comment:
There is a regression reported on the forum, it could be from this line.

Reply:
Hopefully fixed, waiting for other players to test.